torch/_inductor/graph.py

```diff
@@ -411,6 +411,9 @@ def __init__(
         self.named_buffers: dict[str, torch.Tensor] = (
             const_module.named_buffers if const_module else {}
         )
+        self.mutated_named_buffers: OrderedSet[torch.Tensor] = gm.meta.get(
```

Comment:
can or should we re-use `self.mutated_buffers`?

Reply:
`self.mutated_buffers` is only filled as we iterate through the graph, so by the point we check `target in self.mutated_buffers`, `self.mutated_buffers` may not have accurate enough mutation information.